system-tests/services/_scripts/external-volumes/volclean.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
```

Comment:
We can just direct copy-paste something like this with no citation or reference to original source? @MatApple